ui/tests/integration/components/dashboard/client-count-card-test.js

```diff
@@ -64,30 +59,19 @@ module('Integration | Component | dashboard/client-count-card', function (hooks)
     await click('[data-test-refresh]');
   });
 
-  test('it does not query activity for community edition', async function (assert) {
```

Comment:
this whole component is wrapped in an isEnterprise conditional so moved this assertion to the dashboard/overview-test